agent/src/agent/cli/apply.py

```diff
@@ -28,8 +28,7 @@ def apply(work_dir, keep_not_existing):
 
 
 def populate_source_from_file(file):
-    source.check_prerequisites()
-    exceptions = []
+    exceptions = source.check_prerequisites()
```

Comment:
please call `cli.source.check_prerequisites()` here, we shouldn't try to populate sources and pipelines if streamsets has not been set up and that function will raise an exception